### PR TITLE
Feature/dropbox auth

### PR DESCRIPTION
**Pull request by klaseskilson**
Very basic dropbox authentication. User clicks sign in, gets directed to Dropbox's approval page, and then redirected back to Bruse. We then save this information in a session variable.

The information saved is:
- Token
- Name
- Email
- Dropbox user id

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -32,6 +31,15 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
+# Use omniauth for session handling
+gem 'omniauth'
```
Comment (danielronnkvist) [inline Gemfile:35]: This one is included in omniauth-dropbox-oauth2

Comment (klaseskilson) [inline Gemfile:35]: True. However, I think there is a point in keeping this line for clarity.

Comment (klaseskilson) [inline Gemfile:35]: Both removed and then added again per common instructions from Omniauth strategy authors.

**Comment by danielronnkvist** [comment]
I think it's ready for merge!

**Comment by tistatos** [comment]
It is super sexy!!!

**Comment by tkomstadius** [comment]
I think it is nice

**Pull request merged**